cypress/integration/ui_tests/signUp.spec.js

```diff
@@ -5,9 +5,6 @@ describe('Login Test', () => {
     beforeEach(function () {
         cy.viewport('iphone-6');
         cy.visit('https://test.thetreedots.com/');
-    });
-
-    afterEach(() => {
```

Comment:
Why the after each remove?

Reply:
After thinking about it, the clear cookies and cache are better to be done before each test. As it helps the test become cleaner.